app/javascript/packs/admin/editUser.jsx

```diff
@@ -81,15 +88,10 @@ const openModal = async (event) => {
   event.preventDefault();
 
   // Only load Preact if we haven't already.
-  if (!preact) {
-    [preact, { Modal: AdminModal }] = await Promise.all([
```

Comment:
No longer dynamically importing Preact as we use need the `render` for the tag moderation section. The modal remains a dynamic import though.